src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -168,6 +168,8 @@ export const PublishingAccordion = ({
     ? locationState.selectedTicketType === 'PublishingRequest'
     : isDraftRegistration || hasPendingTicket || hasMismatchingPublishedStatus || hasClosedTicket;
 
+  const isWaitingForFileDeletion = isDeletedRegistration && registration.associatedArtifacts.length > 0;
```

Comment:
`associatedArtifacts` kan også ha lenker, som jeg tror ikke slettes når resultatet slettes..? Om det stemmer vil denne da alltid bli `true`, så mistenker at vi egentlig bare vil sjekke for filer her

Reply:
En annen ting med dette er at jeg lurer på om vi kanskje hadde gjort det enklere for oss selv ved å flytte den til over `hasMismatchingPublishedStatus`, og endre til 
```
const hasMismatchingPublishedStatus = mismatchingPublishedStatusWorkflow1 || mismatchingPublishedStatusWorkflow2 || isWaitingForFileDeletion;
```

Da får vi hjelp med at accordion automatisk settes til expanded om man venter på at filer skal slettes. Blir et spørsmål om selve visningen skulle blitt en del av blokken vi har fra før også (linje 222-235), men ikke sikkert det trengs.